.github/workflows/build-nginx-proxy-image.yml

```diff
@@ -0,0 +1,174 @@
+name: Build Nginx Proxy Docker Image
```

Comment:
Maybe call this `Build and Test....` so it's clear in the workflow runs it's getting tests without someone needing to dig into what tests